gradle/testing.gradle

```diff
@@ -99,6 +99,7 @@ def configureTestTask = { propertyPrefix, task ->
         tmpdir.mkdirs()
     }
     task.systemProperties['java.io.tmpdir'] = tmpdir.absolutePath
+    task.systemProperties['junit.jupiter.extensions.autodetection.enabled'] = 'true'
```

Comment:
If I remember correctly this will pick up any extensions from any dependency, right?

Reply:
I believe that would be true, if a dependency has included a test extension in a services file in their jar. I think that would only happen if we relied on a `test` configuration for any of our dependencies. I guess it is somewhat notable for downstream projects that if they relied on our test configuration and had this property set, then they'd get our automatic extensions registered, for better or worse. I'm not too concerned about the exception details one--that seems pretty harmless, in that all it does is generate an extra (useful!) log. The test MDC one I could see being slightly more controversial. If we cared to, we could remove one or both of those and add them to all of our test classes.